packages/office-ui-fabric-react/src/components/DetailsList/DetailsColumn.base.tsx

```diff
@@ -17,7 +17,7 @@ const TRANSITION_DURATION_DROP = 1500; // ms
 const CLASSNAME_ADD_INTERVAL = 20; // ms
 
 /**
- * Component for rendering columns in a {@link DetailsList}.
```

Comment:
@KevinTCoughlin I am removing these references to DetailsList a they don't don't provide any valuable cross reference links currently in either intelisense or website but do create some warnings in docs.microsoft CI. I believe this is happening due to `DetailsList` in this case being just a function that is not supported by `api-extractor` so it can't be a referenced.
![Screen Shot 2019-05-21 at 17 35 44](https://user-images.githubusercontent.com/29514833/58139420-64cbf880-7bef-11e9-88b7-9255549d4504.png)
![Screen Shot 2019-05-21 at 17 38 12](https://user-images.githubusercontent.com/29514833/58139426-6695bc00-7bef-11e9-9d40-c8dd0d77b9f1.png)